packages/terra-clinical-header/src/Header.module.scss

```diff
@@ -1,77 +1,80 @@
+// import themes
+@import './themes/clinical-lowlight-theme/Header.module';
+@import './themes/orion-fusion-theme/Header.module';
+
 :local {
   .flex-header {
-    background-color: var(--terra-clinical-header-background-color, #f4f4f4);
-    border-bottom-color: var(--terra-clinical-header-border-bottom-color, #dedfe0);
-    border-bottom-style: solid;
-    border-bottom-width: 1px;
+    align-items: var(--terra-clinical-header-align-items, flex-start);
+    background-color: var(--terra-clinical-header-background-color, #f9f9f9);
+    border-bottom: var(--terra-clinical-header-border-bottom, 1px solid #d3d4d5);
+    color: var(--terra-clinical-header-color);
     display: flex;
-    height: var(--terra-clinical-header-height, 3.587rem);
     justify-content: space-between;
-    overflow: hidden;
-    padding: var(--terra-clinical-header-padding, 0.714rem);
+    min-height: var(--terra-clinical-header-min-height, 2.92857rem);
+    padding: var(--terra-clinical-header-padding, 0.35714rem 0.35714rem 0.35714rem 0);
     position: relative;
     width: 100%;
 
     .flex-collapse {
-      align-items: center;
+      align-self: var(--terra-clinical-header-flex-collapse-align-self);
       display: flex;
       flex: 0 0 auto;
       height: 100%;
+      margin: var(--terra-clinical-header-flex-collapse-margin);
       max-width: 40%;
       position: relative;
     }
   }
 
+  .subheader {
+    background-color: var(--terra-clinical-header-subheader-background-color, #c8cacb);
+    // color: var(--terra-clinical-header-subheader-color);
+    min-height: var(--terra-clinical-header-subheader-min-height, 2.857rem);
+    padding: var(--terra-clinical-header-subheader-padding, 0.3571rem 0.7143rem);
+  }
+
   .flex-fill {
     align-self: center;
     flex: 1 1 auto;
-    height: 100%;
-    min-width: 0;
     position: relative;
     width: 100%;
   }
 
   .flex-end {
-    align-items: center;
```

Comment:
This needs to be here, otherwise, the buttons are butted up against the top of the box.

Reply:
@jeremyfuksa 

should https://github.com/cerner/terra-clinical/pull/641/files#diff-9f987196835c1f499bb09273c8f5325dR7 be updated from `flex-start` to `center` then?

is action-header incorrect? 
- [action-header header align-items style](https://github.com/cerner/terra-core/blob/master/packages/terra-action-header/src/ActionHeaderContainer.module.scss#L3)
- [action-header flex-end align-self style](https://github.com/cerner/terra-core/blob/master/packages/terra-action-header/src/ActionHeaderContainer.module.scss#L36)

Reply:
They aren't incorrect per se, they're just coded in a way that the inner content is compensating (via line-height, etc.) for what the button doesn't. The text in the header is fine with its alignment style. It's just the buttons that need flexbox vertical alignment.

Reply:
updated: https://github.com/cerner/terra-clinical/pull/641/commits/daeb48597a485300015085b1bccc2a5725d42829

Reply:
![Screen Shot 2020-04-09 09 31 35 AM](https://user-images.githubusercontent.com/726351/78909252-db145c00-7a48-11ea-9637-3412bd9bfa14.png)

Still seeing the buttons butting up against the top.

Reply:
fixed: https://github.com/cerner/terra-clinical/pull/641/commits/0269a181756802e99eac7be85fbbe98734dd096b